* Every os that doesn't match any of the above cases, and has
  /etc/netconfig (like IRIX) will bark that /usr/xpg4/bin/grep is not
  available.  But then again, in prefix we want it to use our own grep
  anyway.

--- configure.in
+++ configure.in
@@ -2116,7 +2116,7 @@
 			;;
 		solaris)
 			if test -f /etc/netconfig; then
-                          if /usr/xpg4/bin/grep -q tcp6 /etc/netconfig; then
+                          if $GREP -q tcp6 /etc/netconfig; then
 				ipv6type=$i
 				ipv6trylibc=yes
                           fi
